client-app/src/admin/tests/Select/SelectTestPanel.js

```diff
@@ -0,0 +1,200 @@
+            placeholder: 'Search restaurants...'
+        })
+    );
+}
+
+function exampleAsync(model, enableWindowed) {
+    const name = (enableWindowed ? 'Windowed' : '') + 'AsyncSelect';
+    return fragment(
+        p(name),
+        label('value: ' + model[name]),
+        select({
+            model,
+            enableWindowed,
+            bind: name,
+            valueField: 'id',
+            labelField: 'company',
+            enableClear: true,
+            selectOnFocus: true,
+            queryFn: (q) => model.queryCustomersAsync(q),
+            optionRenderer: (opt) => customerOption({opt}),
+            placeholder: 'Search customers...'
+        })
+    );
+}
+
+function exampleAsyncCreatable(model, enableWindowed) {
+    const name = (enableWindowed ? 'Windowed' : '') + 'AsyncCreatableSelect';
+    return fragment(
+        p(name),
+        label('value: ' + model[name]),
+        select({
+            model,
+            enableWindowed,
+            bind: name,
+            valueField: 'id',
+            labelField: 'company',
+            enableClear: true,
+            enableCreate: true,
+            selectOnFocus: true,
+            queryFn: (q) => model.queryCustomersAsync(q),
+            optionRenderer: (opt) => customerOption({opt}),
+            placeholder: 'Search customers...'
+        })
+    );
+}
+
+function exampleBig(model, enableWindowed) {
+    const name = (enableWindowed ? 'Windowed' : '') + 'BigSelect';
+    const numOptionsName = (enableWindowed ? 'Windowed' : '') + 'NumOptions';
+    return fragment(
+        p((enableWindowed ? 'Windowed' : '') + 'Select with many options'),
+        label('value: ' + model[name]),
+        select({
+            model,
+            enableWindowed,
+            bind: name,
+            options: function() {
+                let options = [];
+                for (let i = 0; i < model[numOptionsName]; i++) {
+                    options.push(i);
+                }
+                return options;
+            }(),
+            enableClear: false,
+            placeholder: 'Select a number...'
+        }),
+        hbox(
+            label('Number of options: '),
+            numberInput({
+                model,
+                bind: numOptionsName
+            })
+        )
+    );
+}
+
+const customerOption = hoistCmp.factory(
+    ({opt}) => hbox({
+        items: [
+            box({
+                item: opt.isActive ?
+                    Icon.checkCircle({className: 'xh-green'}) :
+                    Icon.x({className: 'xh-red'}),
+                width: 32,
+                paddingLeft: 8
+            }),
+            div(
+                opt.company,
+                div({
+                    className: 'xh-text-color-muted xh-font-size-small',
+                    item: `${opt.city} · ID: ${opt.id}`
+                })
+            )
+        ],
+        alignItems: 'center'
+    })
+);
+
+@HoistModel
+class LocalModel {
+    @bindable
+    Select;
```

Comment:
If this represents a value it should have a name like selectValue.  These look like class names.